app/Http/Controllers/Api/GithubWebhookController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Api;
+
+use App\Handlers\GithubWebhookHandler;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+
+class GithubWebhookController extends Controller
+{
+    public function handle(Request $request): void
+    {
+        $webhookHandler = new GithubWebhookHandler();
```

Comment:
Why handler? Can it be normal service or smth like that?

Reply:
changed it to service